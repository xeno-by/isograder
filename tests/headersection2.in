* Section [0/2]