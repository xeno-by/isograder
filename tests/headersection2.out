EXIT CODE:
1

STDOUT:

STDERR:
error parsing the correction sheet:
line 1: invalid section header format: expected "* <name> [/<max_grade>]"
* Section [0/2]
^

